dev-tools/_functions.sh

```diff
@@ -185,7 +185,7 @@ function ensure_not_root {
 function ensure_root {
     # Check if script is not running as root
     if ! [[ $(id -u) == 0 ]]; then
-        echo "The script ${SCRIPT_NAME} needs root privileges to connect to the docker deamon. It is be automatically restarted with sudo." | print_warning
+        echo "The script ${SCRIPT_NAME} needs root privileges to connect to the docker deamon. It will be automatically restarted with sudo." | print_warning
```

Comment:
```suggestion
        echo "The script ${SCRIPT_NAME} needs root privileges to connect to the docker daemon. It will automatically be restarted with sudo." | print_warning
```
(Not your typo, but I just saw that codespell is complaining)